### DISCUSSION TITLE
Why are some German video titles displayed in English?

### DISCUSSION THREAD

**Discussion by orschiro** [TeamNewPipe/NewPipe#11774] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

See this example. 

Its video title is displayed in English in the news section of NewPipe but displayed correctly in German on the video page itself.

![image](https://github.com/user-attachments/assets/3900bc1e-e2e4-421e-86f6-5212fa33e38b)
<img width="198" alt="image" src="https://github.com/user-attachments/assets/754df88b-e148-4712-a818-5ef608c38d20">

**Comment by ghost** [comment]
Same Thing happend more and more often on my devices (mostly Android Tablet & Fire Sticks).

Some German Channels have now more English Titels than german ones. Titel(s) changed back to German when Video started.

**Comment by ShareASmile** [comment]
#3089
